termbox/src/lib.rs

```diff
@@ -405,6 +383,55 @@ impl Termbox {
             .unwrap();
         }
     }
+
+    fn send_char(&mut self, to_x: u16, to_y: u16, ch: char) {
+        // if the target cell is next to the last cell, then don't move the cursor
+        if (self.last_cursor.0 + 1) == to_x && self.last_cursor.1 == to_y {
```

Comment:
I just realized that this won't work as expected when the last printed char occupies more than one column. See the `cw` variable in `present()`.

This bug actually exists in the original termbox too..

Not sure what's the best way to fix this bug I guess we could maintain a `current_cursor` instead of `last_cursor`, and bump it after sending a character, using the character's visible width.

Reply:
What I'm seeing is that when the last character occupies 2 cells, the next character is not even sent to "send_char". It's being skipped.
I can't look at this right now. I'm in school so I'm just answering between classes. If you want to pick up this, or I can also check it in a few hours.